shared/src/components/Footer/__snapshots__/Footer.test.js.snap

```diff
@@ -64,16 +64,6 @@ exports[`Footer renders 1`] = `
             Legal information
           </a>
         </li>
-        <li
```

Comment:
Maybe it would be good it have a test that asserts that the feedback link shows if analytics is enabled and it's not in debug mode.

Reply:
Thanks, tests added.